pkg/policy/provider.go

```diff
@@ -40,13 +43,35 @@ func newPolicyReconciler(client client.Client, compiler Compiler) *policyReconci
 	}
 }
 
+func (r *policyReconciler) setSortPoliciesFunc() func() []PolicyFunc {
```

Comment:
```suggestion
func setSortPoliciesFunc(policies map[types.NamespacedName]PolicyFunc) func() []PolicyFunc {
```

Reply:
Oh so they are passed by reference, I had the doubt that the state might be stale if I pass it as argument